index.html

```diff
@@ -237,5 +233,61 @@ <h3 class="large">Skills</h3>
         <img src="images/ic_skills.png" class="small-image" alt="">
       </div>
       </div>
+      <footer class="footer">
+        <div class="background-img1">
+        <img src="images/image_geometry_1@2x.png" class="background-image" alt=""> 
+        </div>
+
+       
+          <h3 class="footer-header">Get started</h3>
+
+          <p class="footer-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      
+
+      <div >
+        <form class="form" action="https://formspree.io/f/mqkngogp" method="POST">
+
+          <div class="form-container">
+            <label class="form-label" for="Username"></label>
+            <input type="text" class="name" id="Username" name="Username" placeholder="Your name" maxlength="30" required>
+           </div>
+
+           <div class="form-container">
+             <label class="form-label" for="email"></label>
+            <input type="email" class="email" id="email" name="email"  placeholder="Your email" required>
+            </div>
+
+            <div class="form-container">
+              <label class="form-label" for="message"></label>
+            <textarea class="text-area" id="message" name="Message" required rows="10" maxlength="500" placeholder="Enter your message here..."></textarea>
+            </div>
+
+        <div class="start-btn-div">
+          <button type="submit" class="start-btn-2">Start collaboration</button>
+        </div>
+
+        </form>
```

Comment:
### OPTIONAL

- [ ] Kindly center your form so it is always at the center no matter the mobile screensize, everything looks good at 320px, but as I move to larger mobile screen sizes your form looks aligned to the left, see screenshot below 👍🏿 👍🏿 
![Screenshot_135](https://user-images.githubusercontent.com/69638013/167096251-5a03f777-beed-49bd-ba87-790198bf2616.png)

Reply:
please how would I center the form, don't really understand that, I'm i supposed to use 
 justify-content: center; ?